sdk/core/azure-core/inc/azure/core/datetime.hpp

```diff
@@ -114,7 +114,7 @@ class DateTime final : public _detail::Clock::time_point {
   }
 
   /**
-   * @brief Constructs an instance of `DateTime` from a `time_point`.
+   * @brief Constructs an instance of `%DateTime` from a `time_point`.
```

Comment:
Fyi, there are some other instances of self-referential DateTime links, if you want to fix those.

Reply:
I think there should be a separate effort, I updated instructions for #2299, #2300, and #2304